.github/workflows/release-recorder.yml

```
@@ -16,29 +16,89 @@ jobs:
+        with:
+          go-version-file: "go.mod"
+      - name: Download Go modules
+        run: go mod download
+      - name: Update beta version
+        if: ${{ ! steps.release.outputs.release_created }}
+        run: |
+          cp config_sample.yaml config.yaml
+          GOARCH=amd64 GOOS=linux CGO_ENABLED=0 go build -ldflags '-w -s -buildid=' -o plugnmeet-recorder main.go
+          CURRENT_VERSION=$(./plugnmeet-recorder --version)
+          IFS='-' read -ra VERSION_PARTS <<< "$CURRENT_VERSION"
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)